arviz/tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+"""Configuration for test suite."""
+# pylint: disable=redefined-outer-name
+import os
+import pytest
+
+
+def pytest_addoption(parser):
+    """Definition for command line option to save figures from tests."""
+    parser.addoption("--save", nargs="?", const="test_images",
```

Comment:
@ColCarroll Defaulting to `test_images` is here

Reply:
Oop! Thanks. I'm good merging whenever you/Ari are, then.

Reply:
Wow this is a great idea. Jealous I didn't think of it :)